test/integration/load_stats_integration_test.cc

```diff
@@ -219,8 +219,21 @@ class LoadStatsIntegrationTest : public HttpIntegrationTest,
     do {
       envoy::service::load_stats::v2::LoadStatsRequest local_loadstats_request;
       loadstats_stream_->waitForGrpcMessage(*dispatcher_, local_loadstats_request);
-
+      // Sanity check and clear the measured load report interval.
+      for (auto& cluster_stats : *local_loadstats_request.mutable_cluster_stats()) {
+        const uint32_t actual_load_report_interval_ms =
+            Protobuf::util::TimeUtil::DurationToMilliseconds(cluster_stats.load_report_interval());
+        // Turns out libevent timers aren't that accurate; without this
+        // adjustment we see things like "expected 500, actual 497".
```

Comment:
Is this a failure mode which we expect to be worse on our (slow) CI and ASAN builds?